pandas/tests/reductions/test_reductions.py

```diff
@@ -1060,6 +1060,12 @@ def test_min_max_skipna(self, skipna):
             assert np.isnan(_min)
             assert np.isnan(_max)
 
+    def test_min_max_empty(self):
```

Comment:
move this test to pandas/tests/arrays/categorical/test_analytics.py (there is already a test_min_max) add after

only really need to test the cat itself, don't mind having a Series one here if you want.